xlive/Project_Cartographer.vcxproj.user

```diff
@@ -7,6 +7,13 @@
     <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
     <LocalDebuggerCommandArguments>-nointro -windowed</LocalDebuggerCommandArguments>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='GameRewriteDebug|Win32'">
```

Comment:
please don't add this to the main repository

Reply:
https://github.com/pnill/cartographer/pull/407/commits/15ddad208effb18819e21dedf764dd9bbc93430a#diff-c60806c8d16d9111955b303c7360c44628dff63caab58e858e8c86a84cf0c878L10-L16
Fixed